packages/flutter_test/lib/src/controller.dart

```diff
@@ -1096,36 +1096,42 @@ abstract class WidgetController {
   /// Given a widget `W` specified by [finder] and a [Scrollable] widget `S` in
   /// its ancestry tree, this scrolls `S` so as to make `W` visible.
   ///
-  /// Usually the `finder` for this method should be labeled
-  /// `skipOffstage: false`, so that [Finder] deals with widgets that's out of
-  /// the screen correctly.
+  /// Usually the `finder` for this method should be labeled `skipOffstage:
+  /// false`, so that the [Finder] deals with widgets that are off the screen
+  /// correctly.
   ///
-  /// This does not work when the `S` is long and `W` far away from the
-  /// displayed part does not have a cached element yet. See
-  /// https://github.com/flutter/flutter/issues/61458
+  /// This does not work when `S` is long enough, and `W` far away enough from
+  /// the displayed part does of `S`, that `S` has not yet cached `W`'s element
```

Comment:
looks like there's still an extra "does" in this line?

Reply:
wow that whole sentence was a disaster. fixed.